docs/anforderung/files/project_plan.adoc

```diff
@@ -29,6 +29,9 @@ include::csv/team.csv[]
 ** Tickets aka. WorkItemList:
 *** link:https://github.com/mribrgr/StuRa-Mitgliederdatenbank/issues[Github Issues]
 *** Trello
+** Testen
+*** Selenium
+
 // TODO: Link zum Trelloboard
```

Comment:
Hier fehlt ein Link

Reply:
ich würde das raus löschen, er ist eh nicht in unserer Gruppe und würde das somit sowieso nicht sehen

Reply:
Macht Sinn